js/_api.js

```diff
@@ -20,21 +20,27 @@ export function call(uri, method, payload) {
 		credentials: 'same-origin',
 		method: method,
 		headers: {
-			Accept: 'application/json',
-			'Content-Type': 'application/json'
+			Accept: 'application/json'
 		}
 	};
 
-	if(payload) {
-		var data = new FormData();
-		data.append("json", JSON.stringify(payload));
-		options.body = JSON.stringify(payload);
+	if(method.toLowerCase() === "post") {
+		options.body = new FormData();
```

Comment:
Hrm not making myself clear - the old Dispatcher packed everything into JSON "Details", and then unwrapped them from these Details. I guess that's not really important. I'm happy with whatever you recommend - I just remembered with had a bit of a discussion with Michael whether we submit everything via JSON, or we submit form-encoded, and your opinion was on the side of use-JSON-always :-)